main.cpp

```diff
@@ -147,6 +150,17 @@ void MidiKeyboard::controlMacros(bool macro)
     }
 
 }
+
+struct MidiKeyboardWrapper
+{
+    MidiKeyboardWrapper(MidiKeyboard* MidiKeyboardPointer) : midiKeyboardptr(MidiKeyboardPointer){}
+    ~MidiKeyboardWrapper()
+    {
+        delete midiKeyboardptr;
+    }
+    
+    MidiKeyboard* midiKeyboardptr = nullptr;
```

Comment:
`camelCase` your member variable name.  
Type names begin with capital letters.  variables do not.
- [ ] revise